content/docs/design/visual_story/create_cover_page@ko.md

```diff
@@ -0,0 +1,202 @@
+---
+$title: 커버 페이지를 작성합니다
+---
+
+[TOC]
+
+AMP 스토리의 페이지는 `<amp-story-page>`로 표현합니다. `<amp-story>`
+컴포넌트는 하나 이상의 페이지, 즉 `<amp-story-page>` 컴포넌트를 포함합니다.
+페이지는 이야기의 한 장면이라고 할 수 있습니다. AMP 스토리는 여러개의 페이지
+중에 먼저 작성한 순서대로 사용자에게 페이지를 보여줍니다. 예를 들면, 처음
+기술된 페이지가 사용자가 가장 먼저 보는 페이지입니다.
+
+`<amp-story-page>` 요소를 `<amp-story>`의 자식으로 추가하여 페이지를
+만들어보겠습니다. 그리고 해당 페이지에 고유한 id를 **부여하세요**. 첫 번째
+페이지는 커버가 되는 페이지이므로 `cover`라고 id를 붙여보죠.
+
+
+```html hl_lines="6 7"
+<amp-story standalone
+    title="Joy of Pets"
+    publisher="AMP tutorials"
+    publisher-logo-src="assets/AMP-Brand-White-Icon.svg"
+    poster-portrait-src="assets/cover.jpg">
+   <amp-story-page id="cover">
+   </amp-story-page>
+</amp-story>
+```
+
+커버 페이지의 껍데기가 만들어졌습니다.  하지만 이 정도 가지고는 아직 유효한
+스토리가 아닙니다. 페이지안에는 최소한 하나의 레이어(layer)가 들어있어야 합니다.
```

Comment:
In the English file, there's an image for the layers.  It's missing here.  Please add.